### PR TITLE
[CALC] *.rc: AUTORADIOBUTTONS, ja-JP: Add accel &V

### PR DESCRIPTION
**Pull request by JoachimHenze**
- Tweaks AUTORADIOBUTTONS, code shorter, binary smaller

calc.exe shrinks for GCC8.4.0dbg cfg:
530.432 -> 527.360 bytes

I checked, that there is no change in behavior with those radiobuttons, it is intended (and same on MS Windows Calc that we can not jump from one button to another by arrow-keys here, and also not with tab if any of those radiobuttons has keyboard focus.

- ja-JP: Add missing accel &V in main menu

- in some headers I worked slightly towards the modern style without going all the way yet. Others can do the remaining parts to cover full newstyle **in their own PRs**. I am not greedy for bloatification yet. This also fixes the common ukraiaiaiaiaiaiainian typo

JIRA issue: none

### CODE REVIEW

**Review by whindsaks — APPROVED** [review]

**Comment by whindsaks** [comment]
What is shrinking the size? Is it that DLGITEMTEMPLATE can use the magic WORD instead of a class string.

**Comment by JoachimHenze** [comment]
> What is shrinking the size? Is it that DLGITEMTEMPLATE can use the magic WORD instead of a class string?

The thing that shrinks the size is that we don't bake the "Button" string as often into the binary compared to before.

**Review by katahiromz — APPROVED** [review]

File: base/applications/calc/lang/uk-UA.rc [hunk 1]
```diff
@@ -1,8 +1,7 @@
 /*
- * PROJECT:     ReactOS Calculator
+ * PROJECT:     ReactOS Calc
```
Comment (HBelusca) [inline base/applications/calc/lang/uk-UA.rc:2]: Is this change synchronised with the English translation or the calculator.c files?

**Review by HBelusca — COMMENTED** [review]
I barely see how these changes can in any way influence the final size of the compiled binary file...
Are you sure you didn't have other temporary changes when comparing?

**Review by HBelusca — APPROVED** [review]
Changes look ok otherwise.

**Comment by JoachimHenze** [comment]
> I barely see how these changes can in any way influence the final size of the compiled binary file... Are you sure you didn't have other temporary changes when comparing?

500% sure. And I see the shrinking effect in whatever arbitrary binary that I do these kind of optimizations. I am aware of the effect since more than 10 years already.

File: base/applications/calc/lang/uk-UA.rc (hunk 1 above)
Comment (JoachimHenze) [reply base/applications/calc/lang/uk-UA.rc:2]: It was used in one of the stasm tweaked headers which do reflect the 0.4.15-dev-style already.

**Comment by HBelusca** [comment]
I did some little research, with this example:
```c
    AUTORADIOBUTTON "Radio1",IDC_RADIO1,7,43,180,14,WS_GROUP | WS_TABSTOP
    CONTROL         "Radio2",IDC_RADIO2,"Button",BS_AUTORADIOBUTTON,7,60,180,16
```
and after compilation of the resources, there is _no_ change whatsoever of how both versions are encoded, and they are seen the same way by resource editors:
image
image
(It doesn't matter whether it's in a .res file or in a .exe file.)

You could object that in the resource editor it says: ` CONTROL "Radio2", 1001, BUTTON, ...` instead of ` CONTROL "Radio2", 1001, "Button", ...` ; however manually changing it in the resource editor and recompiling the resource restores the `BUTTON` thing back.

Compiled resource files (.res, or .exe) do not contain the "Button" string (in either ANSI or UTF16 form) in them.

I join in the attached zip file the test examples.
Maybe what you observe is yet another GCC binutils bug where using the different forms cause different things to be compiled...

[test_resources.zip](https://github.com/user-attachments/files/16413397/test_resources.zip)

**Pull request merged**